stl/inc/bit

```diff
@@ -211,18 +211,44 @@ _NODISCARD int _Checked_x86_x64_popcount(const _Ty _Val) noexcept {
 
 
 #if defined(_M_ARM) || defined(_M_ARM64)
+#ifdef __clang__ // workaround
+inline constexpr int _Clang_arm_arm64_countl_zero(const unsigned short _Val) {
+    return __builtin_clzs(_Val);
+}
+
+inline constexpr int _Clang_arm_arm64_countl_zero(const unsigned int _Val) {
+    return __builtin_clz(_Val);
+}
+
+inline constexpr int _Clang_arm_arm64_countl_zero(const unsigned long _Val) {
+    return __builtin_clzl(_Val);
+}
+
+inline constexpr int _Clang_arm_arm64_countl_zero(const unsigned long long _Val) {
+    return __builtin_clzll(_Val);
+}
+#endif // ^^^ __clang__ workaround ^^^
+
 template <class _Ty>
 _NODISCARD int _Checked_arm_arm64_countl_zero(const _Ty _Val) noexcept {
     constexpr int _Digits = numeric_limits<_Ty>::digits;
     if (_Val == 0) {
         return _Digits;
     }
 
+#ifdef __clang__ // workaround
+    if constexpr (sizeof(_Ty) < sizeof(unsigned short)) {
+        return _Clang_arm_arm64_countl_zero(static_cast<const unsigned short>(_Val));

```

Comment:
As an aside, `static_cast<const unsigned short>` is unnecessary (there's no reason to mark the prvalue as `const`) and should just be `static_cast<unsigned short>`.